tests/conftest.py

```diff
@@ -56,6 +57,17 @@ def unvouched_user(stored_users):
     return stored_users['unvouched']
 
 
+@pytest.fixture(scope='session')
+def github_non_nda_user(stored_users):
+    return stored_users['github_non_nda']
+
+
+@pytest.fixture
+def passcode(github_non_nda_user):
+    totp = pyotp.TOTP(github_non_nda_user['secret'])
+    return totp.now()
```

Comment:
Alternatively, you could have `login_with_github` expect the secret, and generate the OTP within that method. I doubt we need to access the `passcode` from anywhere else, and it would mean that a new OTP if a test wanted to log in twice.